src/Symfony/Component/Routing/RouteCollection.php

```diff
@@ -303,4 +328,65 @@ public function addResource(ResourceInterface $resource)
             $this->resources[$key] = $resource;
         }
     }
+
+    /**
+     * Sets an alias for an existing route.
+     *
+     * @param string       $alias The alias to create
+     * @param string|Alias $route The route to alias
+     *
+     * @throws InvalidArgumentException if the id is not a string or an Alias
+     * @throws InvalidArgumentException if the alias is for itself
+     */
+    public function addAlias(string $alias, $route): Alias
+    {
+        if (\is_string($route)) {
+            $route = new Alias($route);
+        } elseif (!$route instanceof Alias) {
+            throw new InvalidArgumentException('$route must be a string, or an Alias object.');
```

Comment:
```suggestion
            throw new InvalidArgumentException(sprintf('"%s" must be a string, or an Alias object.', $route));
```

Reply:
What about ?

```suggestion
            throw new InvalidArgumentException(sprintf(
                'Expected a string or an instance of "%s", but got "%s".',
                Alias::class,
                is_object($route) ? get_class($route) : gettype($route)
            ));
```

Reply:
Hell yeah (It was a fast comment in order to be consistent with others sprintf :sweat_smile: )